src/OrchardCore.Modules/OrchardCore.Title/Drivers/TitlePartDisplay.cs

```diff
@@ -11,48 +10,34 @@ namespace OrchardCore.Title.Drivers
 {
     public class TitlePartDisplay : ContentPartDisplayDriver<TitlePart>
     {
-        private readonly IContentDefinitionManager _contentDefinitionManager;
-
-        public TitlePartDisplay(IContentDefinitionManager contentDefinitionManager)
-        {
-            _contentDefinitionManager = contentDefinitionManager;
-        }
-
         public override IDisplayResult Display(TitlePart titlePart)
         {
             return Initialize<TitlePartViewModel>("TitlePart", model =>
-            {
-                model.Title = titlePart.ContentItem.DisplayText;
-                model.TitlePart = titlePart;
-            })
-            .Location("Detail", "Header:5")
-            .Location("Summary", "Header:5");
+                {
+                    model.Title = titlePart.ContentItem.DisplayText;
+                    model.TitlePart = titlePart;
+                })
+                .Location("Detail", "Header:5")
+                .Location("Summary", "Header:5");
         }
 
-        public override IDisplayResult Edit(TitlePart titlePart)
+        public override IDisplayResult Edit(TitlePart titlePart, BuildPartEditorContext context)
         {
             return Initialize<TitlePartViewModel>("TitlePart_Edit", model =>
             {
                 model.Title = titlePart.ContentItem.DisplayText;
                 model.TitlePart = titlePart;
-                model.Settings = GetSettings(titlePart);
+                model.Settings = context.TypePartDefinition.GetSettings<TitlePartSettings>();
             });
         }
 
-        public override async Task<IDisplayResult> UpdateAsync(TitlePart model, IUpdateModel updater)
+        public override async Task<IDisplayResult> UpdateAsync(TitlePart model, IUpdateModel updater,UpdatePartEditorContext context)
         {
             await updater.TryUpdateModelAsync(model, Prefix, t => t.Title);
 
             model.ContentItem.DisplayText = model.Title;
 
-            return Edit(model);
-        }
-
-        private TitlePartSettings GetSettings(TitlePart titlePart)
-        {
-            var contentTypeDefinition = _contentDefinitionManager.GetTypeDefinition(titlePart.ContentItem.ContentType);
-            var contentTypePartDefinition = contentTypeDefinition.Parts.FirstOrDefault(x => string.Equals(x.PartDefinition.Name, nameof(TitlePart)));
-            return contentTypePartDefinition?.GetSettings<TitlePartSettings>();
+            return Edit(model,context);
```

Comment:
ideal, can you sort the spacing too `Edit(model, context)` and the same on the other prs?